data/docs-redirects.js

```diff
@@ -694,7 +694,7 @@ export const docsRedirects = {
   '/docs/enterprise/integrations/service-now/index.html': '/cloud-docs/integrations/service-now',
   '/docs/enterprise/migrate': '/docs/cloud/migrate',
   '/docs/enterprise/migrate/index.html': '/cloud-docs/migrate',
-  '/docs/enterprise/migrate/workspaces.html': '/docs/cloud/migrate/workspaces.html',
+  '/docs/enterprise/migrate/workspaces.html': '/docs/cloud/migrate/index.html',
```

Comment:
```suggestion
  '/docs/enterprise/migrate/workspaces.html': '/cloud-docs/migrate',
```

I think this is what we'd want, just based on the old URL you're redirecting from.